NEWS.md

```diff
@@ -115,35 +133,41 @@ Standard library changes
   ```julia
   @test isequal(complex(one(T)) / complex(T(Inf), T(-Inf)), complex(zero(T), zero(T))) broken=(T == Float64)
   ```
-  ([#39322])
+  ([#39322]).
 * `@lock` is now exported from Base ([#39588]).
-* The experimental function `Base.catch_stack()` has been renamed to `current_exceptions()`, exported from Base and given a more specific return type ([#29901])
-* Some degree trigonometric functions, `sind`, `cosd`, `tand`, `asind`, `acosd`, `asecd`, `acscd`, `acotd`, `atand` now accept an square matrix ([#39758]).
-* A backslash before a newline in command literals now always removes the newline, similar to standard string
-  literals, whereas the result was not well-defined before. ([#40753])
-* `replace(::String)` now allows multiple patterns to be specified, and they
-  will be applied left-to-right simultaneously, so only one pattern will be
-  applied to any character, and the patterns will only be applied to the input
-  text, not the replacements. ([#TBD])
+* The experimental function `Base.catch_stack()` has been renamed to `current_exceptions()`, exported
+  from Base and given a more specific return type ([#29901]).
+* Some degree trigonometric functions, `sind`, `cosd`, `tand`, `asind`, `acosd`, `asecd`, `acscd`,
+  `acotd`, `atand` now accept a square matrix ([#39758]).
+* `replace(::String)` now accepts multiple patterns, which will be applied left-to-right simultaneously,
+  so only one pattern will be applied to any character, and the patterns will only be applied to the input
+  text, not the replacements ([#40484]).
 
 #### Package Manager
 
```

Comment:
```suggestion
* If a package is `using`/`import`-ed from the `julia>` prompt that isn't found but is available from a registry, a `pkg> add` prompt 
  now offers to install the package into the current environment, parallel precompile it, and continue to load the package ([#39026])
* A new `Manifest.toml` format is now used that captures extensible metadata fields, including the julia version that 
  generated the manifest. Old format manifests are still supported and will be maintained in their original format, unless 
  the user wishes to run `Pkg.upgrade_manifest()` to upgrade the format of the current environment's manifest without re-resolving ([#40765])
* `pkg> precompile` will now precompile new versions of packages that are already loaded, rather than postponing 
  to the next session (The `?` marked dependencies) ([#40345])
* `pkg> rm`, `pin`, and `free` now accept the `--all` argument to call the action on all packages
```